src/FixedDataTableCell.js

```diff
@@ -286,13 +291,16 @@ class FixedDataTableCell extends React.Component {
         </FixedDataTableCellDefaultDeprecated>
       );
     }
-
-    const role = isHeaderOrFooter ? 'columnheader' : 'gridcell';
```

Comment:
We should not remove `role`. keep it for legacy cell as it is, for new cell this could also be added in props.

Reply:
Done